src/AvaTaxClient.php

```diff
@@ -562,6 +595,24 @@ public function getCompany($id, $include)
     }
 
     /**
+     * 
+     *
+     * 
+     *
+     * 
+     * @return FetchResult
+     */
+    public function getCompany()
```

Comment:
getCompany is already declared in class. Method polymorphism is not available in PHP.